data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -76,6 +79,17 @@ find "${ARTIFACTS}/rendered-assets" -name "*kubeconfig*" -print0 | xargs -0 rm
 find "${ARTIFACTS}/rendered-assets" -name "*.key" -print0 | xargs -0 rm
 find "${ARTIFACTS}/rendered-assets" -name ".kube" -print0 | xargs -0 rm -rf
 
+# Collect system information specific to IBM Linux Z (s390x) systems. The dbginfo
+# script is available by default as part of the s390-utils rpm package
+if [ "$arch" == "s390x" ]
+then
+    echo "Gathering dbginfo for the 390x system"
+    mkdir -p "${ARTIFACTS}/node-dbginfo"
+    sudo /usr/sbin/dbginfo.sh -d "${ARTIFACTS}/node-dbginfo"
+    sudo chown -R "${USER}":"${USER}" "${ARTIFACTS}/node-dbginfo"
+    find "${ARTIFACTS}/node-dbginfo" -print0 | xargs -0 chmod a+r
```

Comment:
Can you explain a bit why this is needed? I am confused about why, if we are changing the owner of the collecting info to the user that is collecting the info, then why do we need to allow read permissions to all users? Which other user do we expect to need to read the collected info?